### PR TITLE
Rearrange tasks for uploading archives to maven.

### PR DESCRIPTION
**Pull request by drets**
Rearrange task for uploading archives into maven.

### CODE REVIEW

File: samples/simpleshop/build.gradle [hunk 1]
```diff
@@ -5,11 +5,13 @@ version = '1.0'
...
     mavenCentral()
+    maven {
+        url 'https://github.com/st-kurilin/eventor/raw/master/mvn-repo/'
+    }
 }
 
 dependencies {
-    compile project(':framework')
-
+    compile 'com.eventor:eventor:0.1-SNAPSHOT'
```
Comment (st-kurilin) [inline samples/simpleshop/build.gradle]: Does it work with 0.+ version?

File: build.gradle [hunk 2]
```diff
@@ -39,4 +39,55 @@ task generateReadme << {
...
+        }
+    }
+
+    artifacts {
+        eventorArchives jar
+        eventorArchives sourcesJar
+    }
+}
+
+project(':integration:guice') {
```
Comment (st-kurilin) [inline build.gradle]: It looks like duplication. consider generalizing this logic.

File: samples/simple-cqrs/build.gradle [hunk 3]
```diff
@@ -5,10 +5,13 @@ version = '1.0'
...
+    maven {
+        url 'https://github.com/st-kurilin/eventor/raw/master/mvn-repo/'
+    }
 }
 
 dependencies {
-    compile project(':framework')
+    compile 'com.eventor:eventor:0.1-SNAPSHOT'
     compile project(':test-framework')
     compile project(':integration:guice')
```
Comment (st-kurilin) [inline samples/simple-cqrs/build.gradle:13]: consider changing this dependency ref too.

File: samples/simple-cqrs/build.gradle [hunk 4]
```diff
@@ -5,10 +5,13 @@ version = '1.0'
...
     mavenCentral()
+    maven {
+        url 'https://github.com/st-kurilin/eventor/raw/master/mvn-repo/'
+    }
 }
 
 dependencies {
-    compile project(':framework')
+    compile 'com.eventor:eventor:0.1-SNAPSHOT'
     compile project(':test-framework')
```
Comment (st-kurilin) [inline samples/simple-cqrs/build.gradle:12]: Oh! We have one more artifact to publish.

File: samples/simpleshop/build.gradle (hunk 1 above)
Comment (drets) [inline samples/simpleshop/build.gradle]: No, it does not work well with snapshot version.

File: build.gradle [hunk 5]
```diff
@@ -39,4 +39,29 @@ task generateReadme << {
     }
     out.close()
     inp.close()
+}
+
+[':framework', ':integration:guice', ':test-framework'].each { def projName ->
```
Comment (st-kurilin) [inline build.gradle]: nice

Comment (drets) [inline build.gradle]: thank you!

**Pull request merged**